sdk/communication/communication-chat/src/models/requests.ts

```diff
@@ -2,36 +2,25 @@
 // Licensed under the MIT license.
 
 import { ChatParticipant } from "./models";
-import {
-  CreateChatThreadRequest as RestCreateChatThreadRequest,
-  SendChatMessageRequest as RestSendMessageRequest,
-  AddChatParticipantsRequest as RestAddChatParticipantsRequest
-} from "../generated/src/models";
-export { RestCreateChatThreadRequest, RestSendMessageRequest, RestAddChatParticipantsRequest };
 
-/**
- * An interface representing CreateMessageRequest.
- */
-export interface SendMessageRequest
-  extends Omit<RestSendMessageRequest, "type" | "senderDisplayName"> {}
+export { SendReadReceiptRequest } from "../generated/src/models";
 
-/**
- * Request payload for creating a chat thread.
- */
-export interface CreateChatThreadRequest extends Omit<RestCreateChatThreadRequest, "participants"> {
-  /**
-   * Participants to add to a chat thread.
-   */
+/** Participants to be added to the thread. */
+export interface AddChatParticipantsRequest {
+  /** Participants to add to a chat thread. */
   participants: ChatParticipant[];
 }
 
-/**
- * Thread participants to be added to the thread.
- */
-export interface AddChatParticipantsRequest
-  extends Omit<RestAddChatParticipantsRequest, "participants"> {
-  /**
-   * Participants to add to a chat thread.
-   */
+/** Request payload for creating a chat thread. */
+export interface CreateChatThreadRequest {
+  /** The chat thread topic. */
+  topic: string;
+  /** Participants to be added to the chat thread. */
   participants: ChatParticipant[];
```

Comment:
@DominikMe  `ChatParticipant` is used as input here in `CreateChatThreadRequest` and `AddChatParticipantsRequest`